src/layouts/index.js

```diff
@@ -92,7 +93,12 @@ class Layout extends React.Component {
         <Header 
           headPath = {location.pathname}
           cartLength = {this.state.cartLength} />
-        {children({ ...this.props, eventedLocalStorage: this.eventedLocalStorage })}
+        {
+          children({ ...this.props, 
+          eventedLocalStorage: this.eventedLocalStorage,
+          renderCategoriesListPage: (props) => <CategoryList {...props}/>
```

Comment:
This is the entry layout page. Here we are passing the component to be rendered for category list page as `prop`.